dll/win32/shell32/dialogs/filedefext.h

```diff
@@ -75,7 +75,7 @@ class CFileDefExt :
     static INT_PTR CALLBACK GeneralPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
 	static INT_PTR CALLBACK VersionPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
 	static INT_PTR CALLBACK FolderCustomizePageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
-	BOOL CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, LPDWORD ticks);
+	BOOL CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, LPDWORD ticks);
```

Comment:
nit: this can now be `LPCWSTR` since it is not modified anymore.

Reply:
OK, good point.